develop/tutorials/articles/140-assets/rendering-an-asset.markdown

```diff
@@ -33,6 +33,8 @@ to do to get your asset renderer functioning properly for your asset:
 - Create an asset renderer factory to create an instance of the asset renderer
   for each asset entity.
 
+![Figure 1: The asset renderer factory creates an asset renderer for each asset instance.](../../images/asset-renderer-diagram.png)
```

Comment:
@juliocamarero yeah, good point. I think one arrow would eliminate any confusion. I'll make the change.